pkg/security/scc/byrestrictions.go

```diff
@@ -31,20 +45,20 @@ func pointValue(constraint *securityapi.SecurityContextConstraints) int {
 	// strategies in order of least restrictive to most restrictive
 	switch constraint.SELinuxContext.Type {
 	case securityapi.SELinuxStrategyRunAsAny:
-		points += 4
+		points += runAsAnyUserWeight
```

Comment:
Seems like this be changed to a map like https://github.com/openshift/origin/pull/15116/files#diff-11176a0b63ff747eb428e312553cef58R22 ?